lib/haml_user_tags/compiler.rb

```diff
@@ -11,7 +11,7 @@ def compile(node)
     def convert_user_tag_to_script(node)
       t = node.value
       attributes = t[:attributes]
-      attributes_hashes = t[:attributes_hashes]
+      attributes_hashes = [t[:dynamic_attributes].old] || [{}]
```

Comment:
An array, even if empty, will always be truthy, so should this be like so?

```suggestion
      attributes_hashes = [t[:dynamic_attributes].old || {}]
```

Reply:
@rjpaskin Ah yes, good point. Thanks